external-crates/move/crates/move-compiler/tests/move_2024/parser/top_level_module_double_invalid.exp

```diff
@@ -0,0 +1,8 @@
+error[E02004]: invalid 'module' declaration
+  ┌─ tests/move_2024/parser/top_level_module_double_invalid.move:7:11
+  │
+7 │ module a::n;
+  │           ^ Cannot define a 'module' label form in a file with multiple modules
```

Comment:
Possible to point to the other module? Or too difficult in parsing?

Reply:
Done and done